### PR TITLE
Embed integration into dartpad page

### PR DESCRIPTION
**Pull request by Georgehe4**
Export URLs added into dartpad.

screen shot 2015-07-07 at 10 44 03 am
screen shot 2015-07-07 at 10 43 57 am

@devoncarew

### CODE REVIEW

File: lib/dialogs.dart [hunk 1]
```diff
@@ -91,7 +168,12 @@ class SharingDialog extends DDialog {
     _closeButton = new DButton.button(text: 'Close', classes: 'default');
     _closeButton.onClick.listen((_) => hide());
     _div = new DElement.tag('div')..layoutVertical();
-
+    _exportButton = new DButton.button(text: 'Export', classes: 'default');
```
Comment (devoncarew) [inline lib/dialogs.dart]: Instead of `Export`, what about `Create embeddable link` or `View embeddings options`?

File: lib/dialogs.dart [hunk 2]
```diff
@@ -52,6 +52,82 @@ class AboutDialog extends DDialog {
...
+  ParagraphElement _controls;
+  TextAreaElement _textArea;
+  DButton _cancelButton;
+  DButton _htmlButton;
+  DButton _docButton;
+  DButton _inlineButton;
+  
+  ExportDialog(
+      GistContainer this.gistContainer, GistController this.gistController)
+      : super(title: 'Export') {
```
Comment (devoncarew) [inline lib/dialogs.dart]: `Export` ==> `Embedding Options`?

File: lib/dialogs.dart [hunk 3]
```diff
@@ -52,6 +52,82 @@ class AboutDialog extends DDialog {
...
+    content.setAttr('layout');
+    content.setAttr('vertical');
+
+    _text = content.add(new ParagraphElement());
+    _controls = content.add(new ParagraphElement());
+    _textArea = content.add(new TextAreaElement());
+    _textArea.className = 'sharingSummaryText';
+    _textArea.setAttribute('flex', '');
+
+    _cancelButton = new DButton.button(text: 'Cancel');
```
Comment (devoncarew) [inline lib/dialogs.dart]: Let's remove these three buttons (just have a close button), and automatically generate all three links into the text area when the dialog opens:

```
dart: https://dartpad.dartlang.org/embed-dart.html?id=76567ABC
dart + html: https://dartpad.dartlang.org/embed-html.html?id=76567ABC
dart (minimal): https://dartpad.dartlang.org/embed-inline.html?id=76567ABC
```

File: lib/dialogs.dart (hunk 1 above)
Comment (Georgehe4) [inline lib/dialogs.dart]: Yep. Will change export => embed. Had a brain fart naming the elements.

**Comment by Georgehe4** [comment]
screen shot 2015-07-07 at 1 39 08 pm
Newly updated with suggested modifications.

@devoncarew

**Comment by devoncarew** [comment]
lgtm, nice

**Comment by Georgehe4** [comment]
Merging.

**Pull request merged**